blocks/columns/columns.css

```diff
@@ -166,6 +166,16 @@
   gap: 0;
 }
 
+.columns.contentimage .non-singleton-img-wrapper img {
```

Comment:
I am just wondering if the styles introduced with `contentimage` can be made universal with the columns block itself. thus avoiding introducing a new variant? wdyt?

Reply:
@sdmcraft Thanks for the comment. Two reasons to create the new variant in this case:
1.  The logic in columns.js requires a variant to avoid proceeding wrapImgsInLinks() method.
2. Only part of css can be made universal with the general column block and it requires a regression test which may take much time to make sure it won't affect other pages.

Reply:
Checked the sharepoint docs, and there are some pages using the famous columns block. I'll remove the variant and move the logic to the general version. After that, I'll let all members to have a review if the changing affect their pages.

Reply:
@sdmcraft Completed the change, please check